addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -111,7 +111,7 @@ def _check_before_xml_exporting(self):
 
         # <2.2.1>
         for invoice_line in self.invoice_line_ids:
-            if len(invoice_line.tax_ids) != 1:
+            if not invoice_line.display_type and len(invoice_line.tax_ids) != 1:
```

Comment:
```suggestion
            if invoice_line.display_type:
                continue
            if len(invoice_line.tax_ids) != 1:
```

Reply:
Hmmm, I thought there would be more code below.  There are probably other cases...